src/WebJobs.Extensions.DurableTask/DurableOrchestrationClientBase.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
```

Comment:
>Microsoft.Azure.WebJobs.Extensions.DurableTask [](start = 10, length = 46)

Since this is a public type, let's change the namespace to Microsoft.Azure.WebJobs. I caught this when looking at your change to the sample.